### DISCUSSION TITLE
How many hours on average does a cat sleep each day?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#200] [Galaxy Trivia] [answered]
**Hint:** almost twice as humans

**Comment by muink** [comment, marked as answer]
Cats conserve energy by sleeping more than most animals, especially as they grow older. The daily duration of sleep varies, usually between 12 and 16 hours, with 13 and 14 being the average. Some cats can sleep as much as 20 hours.

**Reply by ParanoidUser** [reply]
You did it! 😋 Congrats on your well deserved [Galaxy Brain](https://github.com/muink?tab=achievements&achievement=galaxy-brain) badge! 🎉
